plugin/build.gradle.kts

```diff
@@ -235,13 +187,13 @@ tasks.withType<Test>().configureEach {
         showStackTraces = true
     }
 
-    retry {
-        val isCiServer = System.getenv().containsKey("CI")
-        if (isCiServer) {
-            maxRetries.set(2)
-            maxFailures.set(10)
-        }
-    }
+//    retry {
+//        val isCiServer = System.getenv().containsKey("CI")
+//        if (isCiServer) {
+//            maxRetries.set(2)
+//            maxFailures.set(10)
+//        }
+//    }
```

Comment:
If the plugin is bundled with the enterprise plugin, this should still work, right?

Reply:
yes, I forgot to re-enable. I will fix